scientific_library/tvb/datatypes/region_mapping.py

```diff
@@ -65,6 +65,23 @@ def from_file(source_file="regionMapping_16k_76.txt"):
         result = RegionMapping()
         result.array_data = reader.read_array(dtype=numpy.int32)
         return result
+    
+    @staticmethod
+    def full_region_mapping(surf, conn):
```

Comment:
Could we maybe receive as input param directly surf.region-_mapping?

Otherwise Here on a theoretical level we have a circular dependency (sim.surface class and RegionMapping)

Reply:
actually it's not the sim.surface class, but the Cortex class that this input param 'surf' represents... 

I know the name is confusing, but I didn't want to change it since in the Simulator class the attribute surface corresponds to an object from the Cortex class.

Reply:
but the remark remains valid... I can change the input param in that case... @maedoc any remarks on this ?